src/celeritas/em/distribution/MuBBEnergyDistribution.hh

```diff
@@ -100,22 +105,17 @@ class MuBBEnergyDistribution
  * Construct with incident and exiting particle data.
  */
 CELER_FUNCTION
-MuBBEnergyDistribution::MuBBEnergyDistribution(Energy inc_energy,
-                                               Mass inc_mass,
-                                               real_type beta_sq,
-                                               Mass electron_mass,
+MuBBEnergyDistribution::MuBBEnergyDistribution(ParticleTrackView const& particle,
                                                Energy electron_cutoff,
-                                               Energy max_secondary_energy)
-    : inc_energy_(value_as<Energy>(inc_energy))
-    , inc_mass_(value_as<Mass>(inc_mass))
-    , total_energy_(inc_energy_ + inc_mass_)
-    , beta_sq_(beta_sq)
+                                               Mass electron_mass)
+    : inc_mass_(value_as<Mass>(particle.mass()))
+    , total_energy_(value_as<Energy>(particle.energy()) + inc_mass_)
```

Comment:
I wonder if `total_energy` should be a `ParticleTrackView` accessor too?

Reply:
True, we use that in several places. I can follow up with a little PR for that.

Reply:
That would be great! 🙇‍♂️